misc-scripts/xref_mapping/XrefParser/MGI_Desc_Parser.pm

```diff
@@ -35,16 +36,14 @@ sub run {
   my $source_id    = $ref_arg->{source_id};
   my $species_id   = $ref_arg->{species_id};
   my $files        = $ref_arg->{files};
-  my $verbose      = $ref_arg->{verbose};
-  my $dbi          = $ref_arg->{dbi};
-  $dbi = $self->dbi unless defined $dbi;
+  my $verbose      = $ref_arg->{verbose} || 0;
```

Comment:
not a big deal in this situation, but it's usually better in this kind of param defaulting to check for definedness instead of truth.
This means I find it safer to use `//` instead of `||`, but it's just a suggestion and personal preference.